### ISSUE TITLE
Bug showing/hiding graphs in combination with candlestick

### ISSUE THREAD

**Issue by healzer** [plotly/plotly.js#2309]
Hi all :)
I have been using PlotlyJS and loving it. Keep up the amazing work!!
Just recently I've been playing with the candlestick plots and noticed a bug when I add a line graph.
After some Googling, I saw that this "bug(?)" can be seen on Plotly's example website:
![candlestick line bug](https://user-images.githubusercontent.com/9488406/35577376-ac5ea8da-05e1-11e8-8442-4ebdb9ee5567.png)

Basically I want to hide the black line when I click on the legend's item, but it's not hiding. It does work when the line is set to "legendonly", and then showing it works -- but hiding again does nothing.

Hope you can help me fix this :)
- Ilya

**Comment by etpinard** [comment]
Thanks for the report.

>  can be seen on Plotly's example website:

Can you be more specific? Pasting the URL would suffice. Thank you.

**Comment by healzer** [comment]
@etpinard yes of course, here you go: https://plot.ly/pandas/candlestick-charts/ 
scroll down to "Out[5]" and try clicking the "Open Vals" label :)
